website/package.json

```diff
@@ -75,14 +76,14 @@
     "ember-cli-sri": "^2.1.1",
     "ember-cli-string-helpers": "^6.1.0",
     "ember-cli-terser": "^4.0.2",
-    "ember-concurrency": "^3.1.1",
+    "ember-concurrency": "^4.0.2",
     "ember-fetch": "^8.1.2",
     "ember-load-initializers": "^2.1.2",
     "ember-math-helpers": "^3.0.0",
     "ember-meta": "^2.0.0",
     "ember-modifier": "^4.1.0",
     "ember-page-title": "^8.0.0",
-    "ember-power-select": "^7.1.2",
+    "ember-power-select": "^8.2.0",
```

Comment:
@alex-ju Does this matter?

Reply:
I see it's been added, resolving the comment then

Reply:
The reason we still have the first-degree dependency on `ember-power-select` is because we have `<PowerSelect>` instances for the overrides. When we retire the overrides we can remove the dependency from `showcase` and `website`.

It was missing from showcase, indeed (was available across workspaces) so I added it in [7af7981](https://github.com/hashicorp/design-system/pull/2022/commits/7af79816edfcc2b8a7073981277fc056973ebec2)